specs/process_open_sockets.table

```diff
@@ -12,7 +12,7 @@ schema([
     Column("remote_port", INTEGER, "Socket remote port"),
     Column("path", TEXT, "For UNIX sockets (family=AF_UNIX), the domain path"),
 ])
-extended_schema(lambda: LINUX() or DARWIN(), [
+extended_schema(lambda: LINUX() or DARWIN() or WINDOWS(), [
```

Comment:
Do we still need this as an extended schema? Considering you're bringing support to Windows we could likely just bring this into the overall schema and get rid of the `extended_schema` bit.

Reply:
I was also wondering about that. I think there's 1 more path for FreeBSD, though, right?

Reply:
Or did you mean that we could always have the `state` column present even if it's blank? I'd be fine with that as well.

Reply:
Ah good call, yeah perhaps leave it as is.